agent/consul/autopilot/autopilot.go

```diff
@@ -22,7 +22,8 @@ type Delegate interface {
 	NotifyHealth(OperatorHealthReply)
 	PromoteNonVoters(*Config, OperatorHealthReply) ([]raft.Server, error)
 	Raft() *raft.Raft
-	Serf() *serf.Serf
+	SerfLAN() *serf.Serf
```

Comment:
Who implements this interface? I can't find much in the codebase with grep, which makes me think it's the customer or external user who implements this?
I think this looks like an API-breaking change, since the type is exported we're changing the interface definition. Is that acceptable? How do you guys usually handle breaking changes to public interfaces?

Apologies if this comment seems irrelevant! Still gaining project context.